.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+contact_links:
+    about: Ask questions about Grails® framework on Github
```

Comment:
I think this line 'about' is leftover from removing discussions record; need to remove it as well.